pino-eventhub.js

```diff
@@ -9,6 +9,9 @@ const crypto = require('crypto')
 const https = require('https')
 const debug = require('debug')('pino-eventhub')
 const Parse = require('fast-json-parse')
+const socketCount = 10
+
+const agent = new https.Agent({keepAlive: true, maxSockets: socketCount})
```

Comment:
this should not be limited to the single module, it should be created within https://github.com/pinojs/pino-eventhub/blob/ff7364f806d6fe905383d3a40c9e4ed7d5c32dba/pino-eventhub.js#L34.